packages/svelte/src/internal/client/dom/elements/transitions.js

```diff
@@ -404,25 +404,13 @@ function animate(element, options, counterpart, t2, on_finish, on_abort) {
 				fill: 'forwards'
 			});
 
-			animation.finished
-				.then(() => {
-					on_finish?.();
-
-					if (t2 === 1) {
-						animation.cancel();
-					}
-				})
-				.catch((e) => {
```

Comment:
The `catch` is necessary to prevent an unhandled rejection [when a transition is aborted](https://svelte-5-preview-52vj098wn-svelte.vercel.app/#H4sIAAAAAAAAE0WQ0W7DIAxFf8Vjk9pKk5K9ZkmlfceyBwamQSWQgelUIf59QCb1AcG1fY-uSUxpg4ENn4lZviIb2Me2sVdG962KcENDWHRw0YtaGYPweqPzbGfS6-Y8QQLFJUIG5d0Kh93Tkec2aNLOHt7rcD0GCcLifmGCl0Cc8Ki4CXiqA6WtohXVAOQuF4PHE6TmpH_PU70bLM927B5J7PgdiYrRWWG0uE5pJ-QWc3_DivATS9PcgVsJYkFxBVoQhLPBGazIHdOQ6VmrFjZXxij1DR4rDXXjKSUZPW_6re_7nM8LGuPGrgwXRuq0yuXvVie10ijZQD5i_sp_rtbO43YBAAA=):

<img width="624" alt="image" src="https://github.com/user-attachments/assets/06dd67f1-14c4-4300-bdfb-f30ee628281b">

Reply:
Ah yeah, I remember now. That sucks :/ can't think of why it causes FF to leak. I've tried making the animation `undefined` inside it but that doesn't help.